lang/cc65/Portfile

```
@@ -12,43 +14,40 @@ long_description \
                 cc65 is a complete cross development package for 65(C)02 \
                 systems, including a powerful macro assembler, a C compiler, \
                 linker, librarian and several other tools.
+license         CC-BY-SA 
 
-# Per http://www.cc65.org/ discontinued support in 2013, moved to github
-# No version tags on github yet
 homepage        http://cc65.github.io/cc65
-distfiles       cc65-sources-${version}.tar.bz2
-use_bzip2       yes
+distfiles       V${version}.tar.gz
 
-master_sites    http://cc65.oldos.net/ \
-                http://bj.spline.de/cc65/ \
-                ftp://ftp.musoftware.de/pub/uz/cc65/ \
-                ftp://ftp.musoftware.de/pub/uz/cc65/old/
+master_sites    git:https://github.com/cc65/cc65/archive
 
-checksums       rmd160  ad728cb61113af4592460bce7d42ca57f6a2fb1a \
-                sha256  a98a1b69d3fa15551fe7d53d5bebfc5f9b2aafb9642ee14b735587a421e00468
+checksums           rmd160  602e4956e4699103ab66dd654bfa53bce0766a85 \
+                    sha256  d14a22fb87c7bcbecd8a83d5362d5d317b19c6ce2433421f2512f28293a6eaab \
+                    size    2254374
```

Align the second column with the rest of the portfile's lines:

```suggestion
checksums       rmd160  602e4956e4699103ab66dd654bfa53bce0766a85 \
                sha256  d14a22fb87c7bcbecd8a83d5362d5d317b19c6ce2433421f2512f28293a6eaab \
                size    2254374
```

---

Should it not be the other way around? Most modern up to date Portfiles indent 20 characters. As does the `post -v checksum -d`  output.

---

done!

---

I don't care what the second column is indented to as long as its consistent within the portfile.